conf/embl_hd.config

```diff
@@ -0,0 +1,46 @@
+params {
+    config_profile_contact     = "gitlab-data-sciences-sci-workflows-nextflow-config-6076-issue-@embl.de"
```

Comment:
Is that really a valid email?! :grimacing:

Reply:
:D yeah it does look weird bu it is a service desk email that will make a gitlab issue on our side.

Reply:
Oky doky :laughing: